mgnify_backlog/mgnify_handler.py

```diff
@@ -334,7 +334,7 @@ def get_request_webin(self, rt_ticket):
     def get_annotation_job_status(self, description):
         return AnnotationJobStatus.objects.using(self.database).get(description=description)
 
-    def get_annotation_jobs(self, run_or_assembly_accessions=None, study_accessions=None, status_description=None,
+    def get_annotation_jobs(self, run_or_assembly_accessions=None, study_accessions=None, status_descriptions=list(),
```

Comment:
```suggestion
    def get_annotation_jobs(self, run_or_assembly_accessions=None, study_accessions=None, status_descriptions=None,
```
Don't use mutable default arguments; these are instantiated when the function is defined and can therefore cause side effects when the function is later called. https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments